lib/IDE/Refactoring.cpp

```diff
@@ -2066,6 +2073,107 @@ bool RefactoringActionSimplifyNumberLiteral::performChange() {
   return true;
 }
 
+static CallExpr *findTrailingClosureTarget(SourceManager &SM,
+                                           ResolvedCursorInfo CursorInfo) {
+  if (CursorInfo.Kind == CursorInfoKind::StmtStart)
```

Comment:
Why are we handling `StmtStart` specifically here?

Reply:
Because `StmtStart` cursor position can't be a part of  `CallExpr`.
We can bail-out early.